File: LeetCode/455-assign-cookies/ex_455.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        int i = 0, j = 0, res = 0;
        sort(g.begin(), g.end());
        sort(s.begin(), s.end());
        while (i < g.size() && j < s.size()) {
            if (g[i] <= s[j]) {
                ++res;
                ++i;
            }
            ++j;
        }
        return res;
    }
};

File: LeetCode/455-assign-cookies/ex_455.java
public class Solution {
    public int findContentChildren(int[] g, int[] s) {
        int i = 0, j = 0, res = 0;
        Arrays.sort(g);
        Arrays.sort(s);
        while (i < g.length && j < s.length) {
            if (g[i] <= s[j]) {
                ++res;
                ++i;
            }
            ++j;
        }
        return res;
    }
}